packages/request-client.js/test/index.test.ts

```diff
@@ -2031,10 +2032,19 @@ describe('request-client.js', () => {
     });
 
     describe('allows overriding the default currencies', () => {
+      const currencyManager = new CurrencyManager(CurrencyManager.getDefaultList());
+      const ETH = currencyManager.from('ETH', 'mainnet');
+
       const requestNetwork = new RequestNetwork({
         signatureProvider: TestData.fakeSignatureProvider,
         useMockStorage: true,
         currencies: [
+          {
+            network: 'mainnet',
+            type: RequestLogicTypes.CURRENCY.ETH,
+            decimals: ETH!.decimals,
+            symbol: ETH!.symbol,
+          },
```

Comment:
```suggestion
      const currencyManager = CurrencyManager.getDefault();
      const ETH = currencyManager.getNativeCurrency(RequestLogicTypes.CURRENCY.ETH, 'mainnet');

      const requestNetwork = new RequestNetwork({
        signatureProvider: TestData.fakeSignatureProvider,
        useMockStorage: true,
        currencies: [
          ETH,
```

Reply:
<img width="660" alt="Screenshot 2023-11-28 at 16 31 57" src="https://github.com/RequestNetwork/requestNetwork/assets/28204154/4acd89b9-32e8-4d0a-a273-909ca70e7891">


Can't do it like that 😢

Reply:
Dang. It would be really nice if this worked. I wonder if it would be worth a future PR to make this usage possible 🤔